packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -6,6 +6,11 @@ export interface ChipProps extends PropsWithChildren {
    */
   readonly ariaLabel?: string;
 
+  /**
+   * The testing id for the chip if necessary. Defaults to `chip-wrapper`.
+   */
+  dataTestID?: string;
```

Comment:
This needs to be a simple pass-through to support Chips (without changing the tests too much). If we wanted to lock down this usage to something specific, we'd have to update all the `chip-wrapper` usages in the tests.